app/controllers/image_controller.rb

```diff
@@ -349,6 +391,10 @@ def process_image(args, upload)
     @image.original_name = "" if @user.keep_filenames == "toss"
     return flash_object_errors(@image) unless @image.save
 
+    add_image_to_observation
```

Comment:
(I love this!  Rubocop really wants to have short methods, but the result of its whining is this!  This is no different from a single long method!  I do the same thing.  It just makes me laugh.  "Oh, you really got me there, rubocop, good catch! " :)